src/Microsoft.DotNet.Darc/src/Darc/Helpers/UxHelpers.cs

```diff
@@ -269,7 +269,8 @@ public static async Task<bool> VerifyAndConfirmBranchExistsAsync(IRemote remote,
             {
                 branch = GitHelpers.NormalizeBranchName(branch);
 
-                await remote.GetDependenciesAsync(repo, branch);
+                var dependencies = await remote.GetDependenciesAsync(repo, branch);
```

Comment:
Why the change in this file?

Reply:
there was a merge conflict, I checked in with the conflict so I had to edit and remove the bad merge.